drivers/gpio/gpio_gd32.c

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2021 Teslabs Engineering S.L.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT gd_gd32_gpio
+
+#include <drivers/gpio.h>
```

Comment:
Please add soc.h in here, gd32f3x0 need it to explicit include the gd32f3x0_gpio.h.

Reply:
done